insights/specs/default.py

```diff
@@ -383,6 +385,30 @@ def du_dirs_list(broker):
     getconf_page_size = simple_command("/usr/bin/getconf PAGE_SIZE")
     getenforce = simple_command("/usr/sbin/getenforce")
     getsebool = simple_command("/usr/sbin/getsebool -a")
+
+    @datasource(Mount, RedHatRelease, HostContext)
+    def gfs2_mount_points(broker):
+        """
+        Function to search the output of ``mount`` to find all the gfs2 file
+        systems.
+        And only run the ``stat`` command on RHEL version that's less than
+        8.3. With 8.3 and later, the command ``blkid`` will also output the
+        block size info.
+
+        Returns:
+            list: a list of mount points of which the file system type is gfs2
+        """
+        gfs2_mount_points = []
+        if (broker[RedHatRelease] and
```

Comment:
The Components IsRhel* is just enhanced and ".minor" is added, would you please change to use the IsRhel* here?

Reply:
@xiangce, sure.